environments/_users/dimagi.yml

```diff
@@ -25,3 +24,4 @@ dev_users:
   absent:
     - astone
     - wpride
+    - gcapalbo
```

Comment:
I think that nclark needs to be removed frrom dimagi-qa

Reply:
It was already in the absent list https://github.com/dimagi/commcare-cloud/blob/master/environments/_users/dimagi-qa.yml#L4

Am I missing any other place?

Reply:
thanks for confirming. misread that before